nginx/templates/_ssl.jinja

```diff
@@ -1,78 +1,78 @@
 {%- if ssl is defined and ssl is mapping and salt['file.file_exists']('/etc/ssl/certs/' ~ ssl.cert) %}
-# Start SSL support for vhost
-# Listen on SSL
+  # Start SSL support for vhost
```

Comment:
You now have a commit message `SBT-585: Add support for enabling/customizing gzip configuration at vhost level` where you change the indentation of a file. Could maybe be cleaner in a separate commit.